packages/graphql/src/classes/Node.ts

```diff
@@ -119,6 +134,19 @@ class Node extends GraphElement {
         this.fulltextDirective = input.fulltextDirective;
         this.auth = input.auth;
         this.queryOptions = input.queryOptionsDirective;
+        this.plural = this.generatePlural();
+    }
+
+    private generatePlural(): string {
+        const name = this.nodeDirective?.plural || this.name;
+
+        const re = /^(_+).+/;
+        const match = re.exec(name);
+        const leadingUnderscores = match?.[1] || "";
```

Comment:
**suggestion (non-blocking)** Consider changing the regex so it always return a valid string with the leading underscores, to remove an extra check:

```suggestion
        const re = /^(_*).+/;
        const match = re.exec(name);
        const leadingUnderscores = match[1];
```

This way, if it doesn't match, it will return a match with an empty string

Reply:
I don't believe it's that simple - `match` can be either a `RegExpExecArray` or `null` - so we need some kind of null assertion at some point.